client/client.go

```diff
@@ -58,6 +58,9 @@ type Client interface {
 	SendPasswordResetEmail(email string) (bool, *model.Response)
 	UpdateUser(user *model.User) (*model.User, *model.Response)
 	UpdateUserMfa(userId, code string, activate bool) (bool, *model.Response)
+	CreateUserAccessToken(userId, description string) (*model.UserAccessToken, *model.Response)
+	RevokeUserAccessToken(tokenId string) (bool, *model.Response)
+	GetUserAccessTokensForUser(userId string, page, perPage int) ([]*model.UserAccessToken, *model.Response)
```

Comment:
Recommend we rename this to `GetUserAccessTokens` to be consistent with the other two APIs.
```suggestion
	GetUserAccessTokens(userId string, page, perPage int) ([]*model.UserAccessToken, *model.Response)
```

Reply:
I think those two are slightly different things, `GetUserAccessTokens` is used for all tokens, not sure if we want something like that. I was on the user oriented side while implementing these features. Maybe we can add this as an extra option or vice versa.

Reply:
Yeah i think that'd be a much better API if we have filter options
`GetUserAccessTokens(filteroptions, page, perPage int) ...`

Is that what you were thinking?

Reply:
Yes, kind of. In the end we can have two commands like:
``` bash
token list <user> #this will only list for a user
token list #this will be listing all user access tokens
```
The thing about renaming; it's bound to the `client4.go` which we *shouldn't` change at this point.

Reply:
o.. got it. 👍